### ISSUE TITLE
Add other sites

### ISSUE THREAD

**Issue by Rexadev** [yogeshwaran01/spotify-playlist-to-youtube-playlist#15]
Spotify, Deezer albums/playlists to YouTube
https://github.com/dbeley/ypc

Deezer to YouTube
https://github.com/Boucanier/deezer_to_youtube

AppleMusic/Spotify/Amazon Music to YouTube Playlists
https://github.com/mnpeep/music2youtube

YouTube to Spotify
https://github.com/piyx/YoutubeToSpotify

NetEase to AppleMusic, YouTube,  Spotify
https://github.com/yueyericardo/Netease-to-Youtube-or-Spotify

**Comment by Rexadev** [comment]
@yogeshwaran01 If you think I am too much into your project. You are right, When I start using a software I go all in and fix it. See  for example foo_openlyrics

**Comment by yogeshwaran01** [comment]
No, not at all. I will always welcome for suggestions and improvements. But I have some other work. So, I unable to work on this project for some time. But I will done it asap.
